apps/ledger-live-mobile/src/components/WebPlatformPlayer/WebViewV2.tsx

```diff
@@ -37,9 +38,12 @@ import prepareSignTransaction from "./liveSDKLogic";
 import { StackNavigatorNavigation } from "../RootNavigator/types/helpers";
 import { BaseNavigatorStackParamList } from "../RootNavigator/types/BaseNavigator";
 import { analyticsEnabledSelector } from "../../reducers/settings";
-import packageInfo from "../../../package.json";
+import getOrCreateUser from "../../user";
 
-const wallet = { name: packageInfo.name, version: packageInfo.version };
+const wallet = {
+  name: "ledger-live-mobile",
```

Comment:
Doesn't want to re-use name from `package.json` anymore?

Reply:
We don't want to import `package.json` in the code and the name in it is not the same

Reply:
ah, I forgot about the inconsistency between desktop and mobile (`ledger-live-desktop` vs `live-mobile` 😑)